content/spin/rust-components.md

```diff
@@ -106,12 +106,12 @@ id = "hello"
 source = "target/wasm32-wasi/release/spinhelloworld.wasm"
 allowed_http_hosts = [ "some-random-api.ml" ]
 [component.trigger]
-route = "/hello"
+route = "/wilcard"
 ```
 
 Running the application using `spin up --file spin.toml` will start the HTTP
 listener locally (by default on `localhost:3000`), and our component can
-now receive requests in route `/hello`:
+now receive requests in route `/wildcard`:
```

Comment:
The bash sample on line 119 also needs to be reflect whatever change we decide to make.

Reply:
Done, in the second commit. Nice catch.